src/lib/webpack-config.js

```diff
@@ -433,6 +435,7 @@ const production = config => addPlugins([
 		minify: true,
 		stripPrefix: config.cwd,
 		staticFileGlobsIgnorePatterns: [
+			/polyfills\.js$/,
```

Comment:
Ah, right. I believe that's the case too 👍 You upgraded my 90% certainty to 95% 😉